tests/prescribed_stokes_solution.prm

```diff
@@ -0,0 +1,181 @@
+# A description of convection in a 2d box. See the manual for more information.
```

Comment:
can you change this line to say something like "test PrescribedStokesSolution" ?